plugins/generic/tinymce

```diff
@@ -1 +0,0 @@
-Subproject commit 949badbccc42f10d732b71df7d0fb1e9b585b565
```

Comment:
oh, no it's not :-( how to proceed? to change and make a new pull request?
THANKS A LOT!!!